Tasks/AzureWebAppOnContainerDeploymentV1/Tests/package-lock.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Can you please make sure each specific version of module must be in thirdpartyNotice .

Reply:
Sure